moveit_ros/moveit_servo/src/servo.cpp

```diff
@@ -561,13 +578,19 @@ TwistCommand Servo::toPlanningFrame(const TwistCommand& command) const
     }
   }
 
-  return TwistCommand{ servo_params_.planning_frame, transformed_twist };
+  return TwistCommand{ planning_frame, transformed_twist };
 }
 
-PoseCommand Servo::toPlanningFrame(const PoseCommand& command) const
+std::optional<PoseCommand> Servo::toPlanningFrame(const PoseCommand& command, const std::string& planning_frame) const
 {
-  return PoseCommand{ servo_params_.planning_frame,
-                      getPlanningToCommandFrameTransform(command.frame_id) * command.pose };
+  const auto planning_to_command_tf_maybe = getPlanningToCommandFrameTransform(command.frame_id, planning_frame);
+  if (!planning_to_command_tf_maybe)
+  {
+    return std::nullopt;
```

Comment:
Print warning or error?

Reply:
Same as above, this is captured in the error from `getPlanningToCommandFrameTransform()`